src/dsp_tools/commands/xmlupload/models/ingest.py

```diff
@@ -107,7 +107,7 @@ def _ingest(self, filepath: Path) -> IngestResponse:
                 elif res.status_code == STATUS_UNAUTHORIZED:
                     raise BadCredentialsError("Bad credentials")
                 else:
-                    user_msg = f"{err} See logs for more details: {logger_savepath}"
+                    user_msg = f"{err} See {warnings_savepath} for more information."
```

Comment:
I have now replace logger_savepath by warnings_savepath in the entire codebase. Because on all occurrences, the message is logged on level warning/error. There is no reference any more to the log file. But I'm not 100% sure if that's what we agreed upon. 

Generally, do you think it makes sense to refer the users to `warnings.log` all the time, or rather to the log file, or perhaps to both? I think it makes most sense to refer only to warnings.log, but I wanted to check with you.

Reply:
I think while we have the warnings file, it is ok to refer only to it. If we delete that, we just should not forget to do a search of the whole repo...

Reply:
Now that we have a variable that is imported, it's impossible to forget :-)